lib/organisation_import/import_trust_data.rb

```diff
@@ -5,6 +5,12 @@ class ImportTrustData < ImportOrganisationData
 
   MEMBERSHIP_TEMP_LOCATION = "./tmp/school-group-membership-data.csv".freeze
 
+  WORDS_TO_UPPERCASE = %w[
+    aces bmat ce csia ebn ekc epa hcat hcuk hisp hti inmat kwest link mat nas own pa pace pact polymat qed qegsmat rmet rnib rsa sdbe seax sen sendat sfaet step tbap tcat timu tvi uk utc vi xxiii
```

Comment:
should `own` be in this list? are these all acronyms?

Reply:
https://www.own-trust.education/

Yes, all acronyms - I checked.